drivers/bluetooth/hci/Kconfig

```diff
@@ -163,6 +163,15 @@ config BT_DRIVER_QUIRK_NO_AUTO_DLE
 	  This has to be enabled when the BLE controller connected is Zephyr
 	  open source controller.
 
+config BT_HCI_SET_PUBADDR
```

Comment:
The only small issue I have here is the consistency with the naming. The new callback is called `set_public_addr`, so maybe it'd make sense for the Kconfig option to be called `BT_HCI_SET_PUBLIC_ADDR`. The other alternative would be to call both `set_pub_addr()` and `BT_HCI_SET_PUB_ADDR`.

Reply:
Renamed to `BT_HCI_SET_PUBLIC_ADDR`, it was bugging me too but not enough to actually do it ;)